CHANGELOG.md

```diff
@@ -0,0 +1,98 @@
+# Changelog
+
+## [Unreleased]
+
+## [v0.1.5] - 2021-01-05
+
+### Added
+
+- Allow support for semicolumn and tabulation in CSV files
+- Collect current distance from OR-Tools
+- Allow initial routes when partitions are required. Note that, this feature is available for preprocessing/partitions but is under development for split based algorithms (`split_solve` and `dichotomous`)
```

Comment:
I think this line should be "[WIP]"

Reply:
The first part is not WIP, the mention for split based algorithms is only present to avoid misinterpretation